src/pages/about.js

```diff
@@ -3,86 +3,88 @@ import { Layout } from "../components/common"
 
```

Comment:
Font is too small and contrast is too low
![image](https://user-images.githubusercontent.com/35622595/130291807-4b7d147a-a8a9-45fb-9112-fde95fdf1008.png)

Reply:
![about](https://user-images.githubusercontent.com/20603515/130300372-a3be55c8-8ba0-4eaa-b666-ccb6a64d5371.png)

better?